README.md

```diff
@@ -16,8 +16,18 @@ Encodes/decodes into lat/lng coordinate pairs. Flip the pairs to be compatible w
 ```js
 var polyline = require('polyline');
 
+// returns an array of lat, lon pairs you can pass to polyline.encode()
+polyline.flip({ 
```

Comment:
Using a GeoJSON `Point` might be a little confusing, most people probably want to encode `LineString` objects?